.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "" # See documentation for possible values
```

Comment:
I would expect us to include `npm` here as part of this PR - were you intending this to be the skeleton without starting updates? Or am I misreading the docs?

Reply:
Whoops! I thought I did that but I clearly did not :D